smarts/core/sumo_road_network.py

```diff
@@ -150,15 +150,19 @@ def from_spec(cls, map_spec: MapSpec):
         # Connections to internal lanes are implicit. If `withInternal=True` is
         # set internal junctions and the connections from internal lanes are
         # loaded into the network graph.
+        # pytype: disable=module-attr
         G = sumolib.net.readNet(net_file, withInternal=True)
+        # pytype: enable=module-attr
```

Comment:
```suggestion
        G = sumolib.net.readNet(net_file, withInternal=True)  # pytype: disable=module-attr
```
If disabling pytype for a single line, perhaps use a single line pytype disable comment.

Reply:
I tried to do that but the `black` formatting caused issues.